core/domain/skill_jobs_one_off_test.py

```diff
@@ -169,3 +169,45 @@ def test_migration_job_converts_old_skill(self):
         expected = [[u'skill_migrated',
                      [u'1 skills successfully migrated.']]]
         self.assertEqual(expected, [ast.literal_eval(x) for x in output])
+
+    def test_migration_job_skips_when_new_structure_editor_is_disabled(self):
+        skill = skill_domain.Skill.create_default_skill(
+            self.SKILL_ID, description='A description')
+        skill_services.save_new_skill(self.albert_id, skill)
+
+        with self.swap(constants, 'ENABLE_NEW_STRUCTURE_EDITORS', False):
```

Comment:
You can probably delete this and similar tests. See #6780.

Reply:
Done